### PR TITLE
Feat/build android

### PR DESCRIPTION
**Pull request by marxeille**

### CODE REVIEW

**Comment by haunv3** [comment]
lgtm

**Comment by haunv3** [comment]
@im6h you want to test ci/cd?

File: .github/workflows/develop-mobile-codepush.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+          restore-keys: |
+            ${{ runner.os }}-yarn-   
+      - name: Install Dependencies
+        if: steps.yarn-cache.outputs.cache-hit != 'true'
+        run: yarn --ignore-platform --ignore-scripts      
+      - name: Install mobile packages        
+        working-directory: packages/mobile
+        run: yarn --silent
+      - name: Install appcenter-cli
+        run: yarn add appcenter-cli
```
Comment (im6h) [inline .github/workflows/develop-mobile-codepush.yaml:55]: @marxeille Can you check this syntax? GA notification it isn't valid

File: .github/workflows/develop-mobile-codepush.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        if: steps.yarn-cache.outputs.cache-hit != 'true'
+        run: yarn --ignore-platform --ignore-scripts      
+      - name: Install mobile packages        
+        working-directory: packages/mobile
+        run: yarn --silent
+      - name: Install appcenter-cli
+        run: yarn add appcenter-cli 
+      - name: Pull packages/background
+        working-directory: packages/background
+        - uses: actions/checkout@v3
```
Comment (im6h) [inline .github/workflows/develop-mobile-codepush.yaml:58]: @marxeille The uses action doesn't need a tab, It is same level with name.

File: .github/workflows/develop-mobile-codepush.yaml (hunk 1 above)
Comment (im6h) [reply .github/workflows/develop-mobile-codepush.yaml:55]: Check this comment, please

**Review by im6h — APPROVED** [review]

**Pull request merged**